### DISCUSSION TITLE
Ordinary Percolation invading phase saturation graph

### DISCUSSION THREAD

**Discussion by L278** [PMEAL/OpenPNM#1957] [Q&A]
I created a simple pore network, where the pores on the left and right of the network were larger than the remaining pores by a magnitude of 10. I set the inlet and outlet as the left and right boundaries.

![image](https://user-images.githubusercontent.com/85620671/127049537-14fdb877-91f1-402a-b149-9aa74bfa6dd2.png)

I was just trying to understand why the saturation of the invading phase decreases at some points, any insight would be appreciated.

**Comment by jgostick** [comment]
My first guess is that you have negative throat lengths due to pores being too large for the lattice (i.e. overlapping).  You can see if this is the case by doing:
* print(geometry['throat.volume']).min()
* Set geometry['throat.volume'] = 0 and return the mip simulation
